scripts/hid_configurator/configurator_cli.py

```diff
@@ -111,6 +112,22 @@ def perform_fwreboot(dev, args):
         print('FW reboot request failed')
 
 
+def perform_led_stream(dev, args):
```

Comment:
Wouldn't it be better to perform command line parameters validation in one place?

Reply:
It could be problematic, since command line parameters depend on selected operation (other for `fwinfo`, `dfu`, `led_stream` etc.). It is why i would prefer to perform check separately.

Reply:
Ok, but provide validation functions appropriate for particula situatio and call them in parse arguments instead of perform - that is not the place where parameters should be validated. After end of parse_params they should be already verified.